app/main/utils.py

```diff
@@ -136,4 +136,26 @@ def create_event_post(event_date,
                    new_value=event_post.val_for_events)
     create_object(event)
 
-    return event_post.id
\ No newline at end of file
+    return event_post.id
+
+
+def get_users_by_division(division):
+    """
+    Query the database for a list of users based on the division passed in
+    :param division: Division to filter by
+    :return: A list of users sorted by last name and filtered by division
+    """
+
+    return Users.query.filter_by(division=division).order_by(Users.last_name).all()
```

Comment:
Given the comment in the views, maybe we can have a dictionary that maps the "page_names" to the division name and use that here?

e.g.
```python
divisions = {
'administration': 'Administration & Human Resources',
...,
...,
...
}
```